hack/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# $1 - Specifies distribution - RHEL7/CentOS7
+# $2 - Specifies Ruby version - 2.0
+
+# Array of all versions of Ruby
+declare -a VERSIONS=(2.0)
+
+OS=$1
+VERSION=$2
+
+if [ -z ${VERSION} ]; then
+  # Build all versions
+  dirs=${VERSIONS}
+else
+  # Build only specified version on Ruby
+  dirs=${VERSION}
+fi
+
+for dir in ${dirs}; do
+  IMAGE_NAME=openshift/ruby-${dir//./}-${OS}
+  echo ">>>> Testing ${IMAGE_NAME}"
+
+  pushd ${dir} > /dev/null
+
+  if [ "$OS" == "rhel7" ]; then
```

Comment:
@bparees Initially I created just one script which handle both - build and test case. But @mfojtik didnt like it and advised me to keep it simple so if anyone will look on it, he gets whats it does. Also this way if anyone will want to create their own STI based images, they will have nice example (and not parsing through some hard to read script)
Does it make sense ?